config/default.yml

```diff
@@ -3770,22 +3852,29 @@ Style/TrailingCommaInArrayLiteral:
   StyleGuide: '#no-trailing-array-commas'
   Enabled: true
   VersionAdded: '0.53'
+  # If `comma`, the cop requires a comma after the last item in an array,
   # but only when each item is on its own line.
   # If `consistent_comma`, the cop requires a comma after the last item of all
-  # non-empty array literals.
+  # non-empty, multiline array literals.
   EnforcedStyleForMultiline: comma
   SupportedStylesForMultiline:
     - comma
     - consistent_comma
     - no_comma
 
+Style/TrailingCommaInBlockArgs:
+  Description: 'Checks for useless trailing commas in block arguments.'
+  Enabled: false
```

Comment:
I believe we could actually enable this cop. It only disallows trailing commas when there are 2 or more block args. One argument may have a trailing comma (for destructuring) or none.

This should be okay, but I don't really consider it that important, so let's maybe leave it off.